returnn/config.py

```diff
@@ -91,6 +92,7 @@ def __init__(
         :param str|None python_epilog_hash: sets a specific hash for the python_epilog
         :param bool hash_full_python_code: By default, function bodies are not hashed. If set to True, the full content
             of python pro-/epilog is parsed and hashed.
+        :param bool sort_dicts: if True, the dicts in the written config file are sorted alphabetically by keys
```

Comment:
Maybe its more interesting to document what happens if it is set to False (i.e. printed in order of insertion)